sdk/resourcemanager/Azure.ResourceManager.Core/README.md

```diff
@@ -14,46 +14,159 @@ Install-Package Azure.ResourceManager.Core -Version 1.0.0-beta.1
 
 ### Prerequisites
 
-* You must have an [Azure subscription](https://azure.microsoft.com/free/)
-
-### Authenticate the Client
+You will need the following values to authenticate to Azure
+
+-   **Subscription ID**
+-   **Client ID**
+-   **Client Secret**
+-   **Tenant ID**
+
+The detailed instructions on how to get and set this values can be found in the [Prerequisites guide](docs/Prerequisites.md). 
+
+### Authentication and Creating Resource Management Client
+
+Once the environment is setup, all you need to do is to create an
+authenticated client. Our default option is to use
+**DefaultAzureCredential** and create an **AzureResourceManagerClient**.  Since all management APIs go through the same endpoint
+you no longer need to create a new client for every resource type like before.
+
+To authenticate to Azure and create an ARM client, simply do the
+following:
+```csharp
+    using Azure.Identity;
+    using Azure.ResourceManager.Core;
+    using System;
+    ...
+    ArmClient armClient = new ArmClient(new DefaultAzureCredential());
+```
+From this code snippet, we showed that in order to interact with Resources, we need to create the top-level client first **AzureResourceManagerClient**
```

Comment:
```suggestion
Once the environment is set up, create an authenticated client. Our default option is to use `DefaultAzureCredential` and create an `AzureResourceManagerClient`. Since all management APIs go through the same endpoint, you no longer need to create a new client for every resource type.

To authenticate to Azure and create an ARM client, do the following:

```csharp
    using Azure.Identity;
    using Azure.ResourceManager.Core;
    using System;
    
    // code omitted for brevity
    
    ArmClient armClient = new ArmClient(new DefaultAzureCredential());
```

From this code snippet, we showed that in order to interact with Resources, we need to create the top-level client first **AzureResourceManagerClient**
```